tests/unittests/client/test_runner.py

```diff
@@ -112,7 +117,14 @@ def function(lhs, sleep):
     return lhs + sleep
 
 
-def new_runner(idle_timeout, n_workers=2, client=None, executor=None, backend="joblib"):
+def new_runner(
```

Comment:
Nit: Type hints.

Also, consider using a `Literal` annotation, e.g. `Literal["joblib", "dask"]` for the `backend` argument, since it's way cooler than just using `str` ;)